api/v1alpha1/database_webhook.go

```diff
@@ -153,6 +130,10 @@ func instanceSpecValidatorForCreate(instance *Instance, allErrs field.ErrorList,
 		allErrs = append(allErrs, field.Invalid(instancePath.Child("databaseInstanceName"), instance.DatabaseInstanceName, "A valid Database Instance Name must be specified"))
 	}
 
```

Comment:
Does the clusterId have a specific syntax? Maybe we can add a webhook validation like we did with the NDB sever.